test/IndexTest.cpp

```diff
@@ -8,6 +8,12 @@
 #include "../src/global/Pattern.h"
 #include "../src/index/Index.h"
 
+ad_utility::LimitedAllocator<Id>& alloc() {
+  static ad_utility::LimitedAllocator<Id> a{
+      ad_utility::makeAllocationState(40000)};
+  return a;
+}
+
 string getStxxlConfigFileName(const string& location) {
   std::ostringstream os;
   os << location << ".stxxl";
```

Comment:
This reminds me of a naming problem that has been bothering me for a long time:

The stxxl disk file is called <basename>-stxll.disk or something like that. It should be called <basename>.stxxl.disk

Reply:
You might be right, but this is not the place for this.
I am always happy when the automatic generation of the stxxl files works. You could comment this elsewhere (ticket etc?)

Reply:
OK